src/sentry/runner/commands/devservices.py

```diff
@@ -65,9 +82,11 @@ def _client() -> ContextManager[docker.DockerClient]:
             if DARWIN:
                 if USE_COLIMA:
                     click.echo("Attempting to start colima...")
+                    gitroot = _gitroot()
                     subprocess.check_call(
                         (
-                            "devenv",
+                            # explicitly use repo-local devenv, not the global one
+                            f"{gitroot}/.venv/bin/devenv",
```

Comment:
better would be `sys.executable -m devenv` -- but we never finished the migration to `.devenv` or whatever for the configuration